tests/test_irodspath.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import PurePosixPath, PureWindowsPath
+
+from pytest import mark
+
+from ibridges import IrodsPath
+
+
+class MockIrodsSession:
+    zone = "testzone"
+    user = "testuser"
+    host = "test.host.nl"
+    server_version = "test_version"
+    port = 9876
+
+    def home(self):
+        return "/"+self.zone+"/home/"+self.user
+
+    def cwd(self):
```

Comment:
We do not need the cwd any longer, right?

Reply:
No, I removed it.